### PR TITLE
chaos(engine): fix kube config can't find in root user

### LINKED ISSUE #6986: Pod logs in chaos test are not uploaded
### What did you do?

ref: https://github.com/pingcap/tiflow/actions/runs/3003923489

### What did you expect to see?

_No response_

### What did you see instead?

The Artifacts only contain kind logs, without any `server-master` or `executor` log.

```bash
kubectl get pods --no-headers -o custom-columns=":metadata.name"|grep -E "server-master|executor"|xargs -I{} sudo kubectl cp {}:/log ./logs || true
  kind export logs ./logs/kind --name dataflow-engine-cluster
  sudo chown -R runner ./logs
  shell: /usr/bin/bash -e {0}
  env:
    CFG_BASE6[4](https://github.com/pingcap/tiflow/runs/8217805325?check_suite_focus=true#step:21:4): YXBpVmVyc2lvbjogY2hhb3MtbWVzaC[5](https://github.com/pingcap/tiflow/runs/8217805325?check_suite_focus=true#step:21:5)vcmcvdjFhbHBoYTEKa2luZDogUG9kQ2hhb3MKbWV0YWRhdGE[6](https://github.com/pingcap/tiflow/runs/8217805325?check_suite_focus=true#step:21:6)CiAgbmFtZTogcG9kLWZhaWx1cmUtZGF0YWZsb3cKICBsYWJlbHM6CiAgICBhcHA6IHBvZC1mYWlsdXJlLWRhdGFmbG93CnNwZWM6CiAgYWN0aW9uOiBwb2QtZmFpbHVyZQogIG1vZGU6IG9uZQogIGR1cmF0aW9uOiAiMzBzIgogIHNlbGVjdG9yOgogICAgcG9kczoKICAgICAgZGVmYXVsdDogIyBkZWZhdWx0IG5hbWVzcGFjZQogICAgICAgIC0gY2hhb3Mtc2VydmVyLW1hc3Rlci0wCiAgICAgICAgLSBjaGFvcy1zZXJ2ZXItbWFzdGVyLTEKICAgICAgICAtIGNoYW9zLXNlcnZlci1tYXN0ZXItMgogICAgICAgIC0gY2hhb3MtZXhlY3V0b3ItMAogICAgICAgIC0gY2hhb3MtZXhlY3V0b3ItMQogICAgICAgIC0gY2hhb3MtZXhlY3V0b3ItMgogICAgICAgIC0gY2hhb3MtZXhlY3V0b3ItMwogIHNjaGVkdWxlcjoKICAgIGNyb246ICJAZXZlcnkgMm0iCg==
The connection to the server localhost:[8](https://github.com/pingcap/tiflow/runs/8217805325?check_suite_focus=true#step:21:8)080 was refused - did you specify the right host or port?
The connection to the server localhost:8080 was refused - did you specify the right host or port?
The connection to the server localhost:8080 was refused - did you specify the right host or port?
The connection to the server localhost:8080 was refused - did you specify the right host or port?
The connection to the server localhost:8080 was refused - did you specify the right host or port?
The connection to the server localhost:8080 was refused - did you specify the right host or port?
The connection to the server localhost:8080 was refused - did you specify the right host or port?
Exporting logs for cluster "dataflow-engine-cluster" to:
./logs/kind
```

### Versions of the cluster

Dataflow Engine version (run `tiflow version`):

master@https://github.com/pingcap/tiflow/commit/4a97c4527a56d531d0abd9f9f599ac9006dc8275

### PR DESCRIPTION
**Pull request by amyangfei**
### What problem does this PR solve?

Issue Number: close #6986

### What is changed and how it works?

Can't reproduce in local environment, try to debug it.

- Use concete container name when executing `kubectl exec`
- Use normal user to execute `kubectl cp`

#### Tests 

 - Chao test

### Release note 

```release-note
None
```

### CODE REVIEW

**Comment by amyangfei** [comment]
Chaos test runs successfully. Logs of both `server-matser` and `executor` are uploaded.

https://github.com/pingcap/tiflow/actions/runs/3004855743

```bash
Run mkdir ./logs
tar: removing leading '/' from member names
tar: removing leading '/' from member names
tar: removing leading '/' from member names
tar: removing leading '/' from member names
tar: removing leading '/' from member names
tar: removing leading '/' from member names
tar: removing leading '/' from member names
Exporting logs for cluster "dataflow-engine-cluster" to:
./logs/kind
```

**Review by CharlesCheung96 — APPROVED** [review]

File: .github/workflows/dataflow_engine_chaos.yaml [hunk 1]
```diff
@@ -221,14 +221,14 @@ jobs:
         run: |
           # Add a delay if test fails, to check whether the cluster can recover after chaos is removed
           sleep 60
-          kubectl get pods --no-headers -o custom-columns=":metadata.name"|grep -E "server-master"|xargs -I{} kubectl exec -it {} -- wget http://127.0.0.1:10240/debug/pprof/goroutine?debug=2 -O /log/{}_goroutines.log || true
-          kubectl get pods --no-headers -o custom-columns=":metadata.name"|grep -E "executor"|xargs -I{} kubectl exec -it {} -- wget http://127.0.0.1:10241/debug/pprof/goroutine?debug=2 -O /log/{}_goroutines.log || true
+          kubectl get pods --no-headers -o custom-columns=":metadata.name"|grep -E "server-master"|xargs -I{} kubectl exec -i -c server-master {} -- wget http://127.0.0.1:10240/debug/pprof/goroutine?debug=2 -O /log/{}_goroutines.log || true
```
Comment (lonng) [inline .github/workflows/dataflow_engine_chaos.yaml:224]: Can we rename the `server-master` to `master` and make the names symetric?

**Review by lonng — APPROVED** [review]

File: .github/workflows/dataflow_engine_chaos.yaml (hunk 1 above)
Comment (amyangfei) [reply .github/workflows/dataflow_engine_chaos.yaml:224]: Agreed, added an issue to track the rename. https://github.com/pingcap/tiflow/issues/6997

**Pull request merged**